### PR TITLE
cronjob : remove sinusbot & improve ytdl

### PR DESCRIPTION
**Pull request by zerefdev**
Removed sinusbot cronjob since sinusbot does not support "-update".
Improved ytdl cronjob; echo ->printf (multilines) BASH&PATH to avoid 

> /bin/sh: 1: youtube-dl: not found

### CODE REVIEW

File: sinusbot_installer.sh [hunk 1]
```diff
@@ -927,7 +924,7 @@ if [ "$YT" == "Yes" ]; then
     redMessage "Cronjob already set for YT-DL updater"!
   else
     greenMessage "Installing Cronjob for automatic YT-DL update..."
-    echo "0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename >/dev/null" >>/etc/cron.d/ytdl
+    printf "SHELL=/bin/sh\nPATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin\n0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename > /dev/null 2>&1" >>/etc/cron.d/ytdl
```
Comment (mxschmitt) [inline sinusbot_installer.sh]: Why are you setting there the `PATH` environment variables for this cronjob manually?
Instead I would only set an absolute path for the youtube-dl binary.

**Review by mxschmitt — CHANGES_REQUESTED** [review]

File: sinusbot_installer.sh [hunk 2]
```diff
@@ -927,7 +924,7 @@ if [ "$YT" == "Yes" ]; then
     redMessage "Cronjob already set for YT-DL updater"!
   else
     greenMessage "Installing Cronjob for automatic YT-DL update..."
-    echo "0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename >/dev/null" >>/etc/cron.d/ytdl
+    printf "SHELL=/bin/sh\nPATH=/usr/local/bin/\n0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename > /dev/null 2>&1" >>/etc/cron.d/ytdl
```
Comment (irgendwr) [inline sinusbot_installer.sh:927]: As @mxschmitt mentioned: Why are you setting SHELL and PATH? There is also no need for `printf`. Just use an absolute path:
```bash
echo "0 0 * * * $SINUSBOTUSER /usr/local/bin/youtube-dl -U --restrict-filename > /dev/null 2>&1" >>/etc/cron.d/ytdl
```

File: sinusbot_installer.sh [hunk 3]
```diff
@@ -912,11 +912,8 @@ if [ "$INSTALL" == "Inst" ]; then
 fi
 
 if [[ -f /etc/cron.d/sinusbot ]]; then
-  redMessage "Cronjob already set for SinusBot updater"!
-else
-  greenMessage "Installing Cronjob for automatic SinusBot update..."
-  echo "0 0 * * * $SINUSBOTUSER $LOCATION/sinusbot -update >/dev/null" >>/etc/cron.d/sinusbot
-  greenMessage "Installing SinusBot update cronjob successful."
+  rm /etc/cron.d/sinusbot
+  redMessage "Sinusbot does not support the 'update' option"!
```
Comment (irgendwr) [inline sinusbot_installer.sh:916]: This is doesn't need to be printed to the user.

**Review by irgendwr — CHANGES_REQUESTED** [review]

File: sinusbot_installer.sh (hunk 2 above)
Comment (Zerefdev) [reply sinusbot_installer.sh]: it doesn't work, Cron clears the whole environment..
This what users get
 [CODE]/bin/sh: 1: youtube-dl: not found [/CODE]

**Comment by irgendwr** [comment]
🙁

**Pull request closed without merging**